WordPress/src/main/java/org/wordpress/android/ui/reader/viewmodels/ReaderPostListViewModel.kt

```diff
@@ -249,6 +261,37 @@ class ReaderPostListViewModel @Inject constructor(
                 ))
     }
 
+    /**
+     * Handles reblog button action
+     *
+     * @param post post to reblog
+     */
+    fun onReblogButtonClicked(post: ReaderPost) {
+        val selectedSiteId = appPrefsWrapper.getSelectedSite()
+        val selectedSite: SiteModel = siteStore.getSiteByLocalId(selectedSiteId)
+        when (siteStore.visibleSites.size) {
+            0 -> _reblogAction.value = Event(NoSite)
+            1 -> _reblogAction.value = Event(PostEditor(selectedSite, post))
+            else -> _reblogAction.value = Event(SitePicker(selectedSite, post))
+        }
+    }
+
+    /**
+     * Handles site selection
+     *
+     * @param site selected site to reblog to
+     */
+    fun selectedSiteToReblog(siteLocalId: Int) {
```

Comment:
Wdyt on keeping the same `event style` naming here? Something like `onReblogSiteSelected`?

Reply:
I agree. Fixed 074a7a0e1cdb0cb89e06c885b76adfda130fb11c